policyengine_us/parameters/gov/states/ma/dta/tcap/tafdc/eligibility/income_limit/additional_person.yaml

```diff
@@ -0,0 +1,18 @@
+description: Massachusetts increases the eligibility limit for the Transitional Aid to Families with Dependent Children by this amount for each additional household member.
+values:
+  2021-01-01: 116
+  2022-01-01: 126
+  2023-01-01: 139
+metadata:
+  unit: currency-USD
+  period: month
+  label: Massachusetts TAFDC additional person eligibility limit increases
+  reference:
```

Comment:
no statutes/regs for this, at least structurally?

Reply:
No, the [regs](https://www.law.cornell.edu/regulations/massachusetts/106-CMR-704-420) only point to mss.gov 
![image](https://github.com/user-attachments/assets/ef5df971-2ac6-47dc-994c-76ef1a3224dc)